README.md

```diff
@@ -283,7 +284,7 @@ The next steps we plan to work on are:
 - [Starlark](https://docs.bazel.build/versions/master/skylark/language.html) is
     the language of [Bazel](https://bazel.build/), which is a dialect of
     [Python](https://www.python.org/). It does not have types and recursion is
-    forbidden, making it not Turing-complete.
+    forbidden, making it not turing-complete.
```

Comment:
I actually agree with this change. I don't like capitalisation of proper nouns when they become part of compound words, especially as they get more lexicalised. However
(a) formatting standards seem to disagree with us
(b) this would have to be changed consistently through the document